draft-ietf-mls-architecture.md

```diff
@@ -2014,6 +2014,28 @@ the protocol to securely implement the specification, which remains non-trivial.
 > the MLS clients from physical compromise. In such settings, HSMs and secure
 > enclaves can be used to protect signature keys.
 
+# No Protection against Replay by Insiders
+
+MLS does not provide protections against replay of group messages by members 
+of the group.  Applications for whom replay is an important risk should apply 
+mitigations at the application layer, as discussed below.
```

Comment:
```suggestion
MLS does not protect against one group member replaying a PrivateMessage sent by another
group member within the same epoch that the message was originally sent. Similarly, MLS
does not protect against the replay (by a group member or otherwise) of a PublicMessage
within the same epoch that the message was originally sent. Applications for
whom replay is an important risk should apply mitigations at the application layer, as
discussed below.
```